crates/ark/src/lsp/completions/completion_context.rs

```diff
@@ -13,12 +13,14 @@ use crate::lsp::completions::sources::composite::pipe::find_pipe_root;
 use crate::lsp::completions::sources::composite::pipe::PipeRoot;
 use crate::lsp::document_context::DocumentContext;
 use crate::lsp::state::WorldState;
+use crate::treesitter::node_find_containing_call;
 
 pub(crate) struct CompletionContext<'a> {
     pub(crate) document_context: &'a DocumentContext<'a>,
     pub(crate) state: &'a WorldState,
     parameter_hints_cell: OnceCell<ParameterHints>,
     pipe_root_cell: OnceCell<Option<PipeRoot>>,
+    is_in_call_cell: OnceCell<bool>,
```

Comment:
New field that is useful to multiple completion sources.

*I think I'm going to add the call node itself as a field, so perhaps not quite done here.*

Reply:
Yea I'm expecting to see something like `OnceCell<Option<Node>>` here in the end

Reply:
Done!